controllers/helmchart_controller_test.go

```diff
@@ -2038,6 +2029,198 @@ func TestHelmChartReconciler_notify(t *testing.T) {
 	}
 }
 
+func TestHelmChartReconciler_reconcileSource_authStrategy(t *testing.T) {
```

Comment:
This test is testing auth strategy for OCI only. Let's reflect that in the name.